### ISSUE TITLE
This repo is abandoned (but still gets lots of issue reports and PR's)

### ISSUE THREAD

**Issue by peterdew** [twitter/typeahead.js#1676]
This repo is abandoned now for 2 year but is still popular and people implement it a lot in their solutions. So was I. 

People send issue requests almost daily, and even PR's to help maintaining and put hours of effort before they become aware that there is no follow up at all. 

Please make certain that this is the case in the title, or even transfer ownership please, to avoid this waste!

@jharding, @vskarich or @timtrueman, can you do something about this! It is totally cool that you stop spending your effort, but it would be very helpfull to transfer ownership then, or at least mark the repo as abandoned and point to a fork were maintenance continues. ([this one ](https://github.com/corejavascript/typeahead.js)for example?)

**Comment by peterdew** [comment]
@jlbooker Hi Jeremy, 
I see, you are contributor here, and also a contributor at corejavascript/tyepahead.js. Can you help us out here?

**Comment by jlbooker** [comment]
Hi @peterdew, I don't have any permissions in this repo, so not much I can do. Sorry! Do try the [corejavascript/typeahead.js](https://github.com/corejavascript/typeahead.js) project instead! (I am a maintainer there.)

**Comment by peterdew** [comment]
I sended this to Github as a support request: 
the Twitter/typeahead-repo is abandoned now for 2 year but is still popular and people implement it a lot in their solutions. So was I.

People send issue requests almost daily, and even PR's to help maintaining and put hours of effort before they become aware that there is no follow up at all.

I created an issue to try to reach one of the maintainers but nobody responds. 

What is the prefered way to fix this kind of problems? There is a fork that continues the maintenance, but that is hard to find, and people tend to stick to the 'official' repo until the become aware that that is useless.

Kan you reach ou tto twitter to ask if they are willing to continue maintenance,  change ownership or put the repo to readonly with a pointer to the maintained fork? 
Original repo:       https://github.com/twitter/typeahead.js 
Maintained fork: https://github.com/corejavascript/typeahead.js

**Comment by peterdew** [comment]
I got this response from GitHub support: 
There are a few options in this situation:

> Ask the maintainers to add new active maintainers
> Ask the maintainers to link to the currently maintained fork as the new main project
> The owner of corejavascript/typeahead.js can ask us to detach their fork and turn it into a standalone repository so that it can be more than a fork of this unmaintained project.
> It looks like the maintainers have listed contact details in the readme. Have you tried reaching out to them?

The contact-data involves only the twitter-accounts. I sended a tweet to all 3 of the current maintainers, and also to the twitter open source software twitteraccount. @TwitterOSS Didn't get any responses yet. 

Any other ideas?

**Comment by peterdew** [comment]
Correction: @vskarich just responded and he will take a look what he can do about it.

**Comment by peterdew** [comment]
Hi @vskarich, did you have time to take a look yet? Would be great if you could resurrect this repo or make it read-only and point it to corejavascript/typeahead.js.

Would Twitter be interested in activate maintenance again? Do you still have contacts there to check that? Or maybe you can simply transfer ownership to the current owner of corejavascript/typeahead.js.

**Comment by jesperronn** [comment]
As I understand it, the `corejavascript/typeahead.js` is already both:

* a clone of this repos
* actively being maintained

So, it doesnt make sense to transfer ownership. 

Instead, please update the README file of this project with the relevant info. And perhaps commit a deletion of all other files. This will help people using the new location at https://github.com/corejavascript/typeahead.js

**Comment by peterdew** [comment]
True, it only makes sense if the current owners are not motivated to do al kind of changes and leave that to the current maintainers.
It would also be nice when ownership of the nuget-package that is now an old release from twitter/typeahead could be managed by the owners of the active github-repo!
See [transfering-package-ownership](https://blog.nuget.org/20130205/managing-package-owners.html#transfering-package-ownership)
and [orphaned-packages](https://blog.nuget.org/20130205/managing-package-owners.html#orphaned-packages)

**Comment by peterdew** [comment]
Hi @vskarich, a few weeks ago you planned to take a look at this repo. What are your findings till now? Can you help us out?

**Comment by Cherna** [comment]
Thanks, I was exactly looking for this information. Is there no one left that can modify the readme to point to corejs fork of this repo?

**Comment by peterdew** [comment]
The only person I got in contact with was vskarich. If he do not respond anymore we can ask GitHub support to modify the repo for us. I'll do that in a few weeks.

**Comment by voorbeeldgebruiker** [comment]
I've send another mail to GitHub, hopefully they can help us out here.

**Comment by peterdew** [comment]
I've send another mail to GitHub, hopefully they can help us out here.

**Comment by kamilwylegala** [comment]
Hey @peterdew, Did you get any response?

**Comment by peterdew** [comment]
@kamilwylegala   Yes, I did. A disappointing one. That they can't be of any help, witch I not understand because they can easily make the necessary changes.

**Comment by cdalexndr** [comment]
https://github.com/corejavascript/typeahead.js seems abandoned too, last commit was on Sep 25, 2017

**Comment by jlbooker** [comment]
We're still alive over there, just hasn't been much activity lately.

**Comment by cdalexndr** [comment]
Here's another fork that seems to be more active: https://github.com/algolia/autocomplete.js/graphs/code-frequency
It's a pity that the development efforts are not focused on a single repository.

**Comment by dandelionred** [comment]
It is a shame twitter acts this way with typeahead. We should reach Elon Musk on this matter.

**Comment by sokollondon** [comment]
> Here's another fork that seems to be more active: https://github.com/algolia/autocomplete.js/graphs/code-frequency
> It's a pity that the development efforts are not focused on a single repository.

algolia/autocomplete.js seems not free because it requires getting 'YourApplicationID', 'YourSearchOnlyAPIKey' in algolia.com

Therefore **https://github.com/corejavascript/typeahead.js** is the **best choice for today**.
